### DISCUSSION TITLE
Apply GitHub Education

### DISCUSSION THREAD

**Discussion by Asher-hss** [community/community#144843] [GitHub Education]
### Hi everyone,

As a Georgia Tech student, I tried applying for this service over and over again, but I kept getting rejected. I attempted to submit more documents to prove my identity, but it was of no use.

**Comment by FeiSuoerDe** [comment]
As a lesson from experience, currently I succeeded only after nineteen attempts.

**Reply by Asher-hss** [reply]
same picture？

**Reply by FeiSuoerDe** [reply]
yep，I used my school profile and filmed it repeatedly
